velox/functions/prestosql/window/tests/LeadLagTest.cpp

```diff
@@ -184,6 +184,26 @@ TEST_P(LeadLagTest, ignoreNullsInt64Offset) {
   assertResults(fn(fmt::format("c0, {} IGNORE NULLS", largeOffset)));
 }
 
+TEST_P(LeadLagTest, ignoreNullsZeroOffset) {
+  auto data = makeRowVector({
+      // Values.
+      makeNullableFlatVector<int32_t>(
+          {1, std::nullopt, 2, std::nullopt, std::nullopt}),
+      // Offsets.
+      makeFlatVector<int64_t>({0, 0, 0, 0, 0}),
+  });
+  createDuckDbTable({data});
+
+  auto assertResults = [&](const std::string& functionSql) {
+    auto queryInfo = buildWindowQuery({data}, functionSql, "order by c0", "");
+    SCOPED_TRACE(queryInfo.functionSql);
+    assertQuery(queryInfo.planNode, queryInfo.querySql);
+  };
+
+  assertResults(fn("c0, 0 IGNORE NULLS"));
```

Comment:
Add tests for fn("c0, 0"), fn("c0, c1") (so basically even test without IGNORE NULLs since we have now changed those code-paths as well) and also add the same tests with the column without NULL values.